### PR TITLE
Feature/1422/link to test parameter file

### PR DESCRIPTION
**Pull request by garyluu**
For review only, changing base to hotfix later

### CODE REVIEW

File: dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java [hunk 1]
```diff
@@ -138,6 +138,25 @@ private void toolsIdVersionsVersionIdTypeDescriptorRelativePathNormal() throws E
         assertDescriptor(SUPPORT.getObjectMapper().writeValueAsString(responseObject));
     }
 
+    // Use relative-paths endpoint to return of the test parameter file
+    @Test
+    public void toolsIdVersionsVersionIdTypeDescriptorRelativePathTestFile() throws Exception {
+        Response response = checkedResponse(
+            basePath + "tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/PLAIN_TEST_CWL_FILE/descriptor/%2Ftest.cwl.json");
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java]: Hmmm, might be a miscommunication on my end.

 I think I'd prefer to use 
`tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/CWL/descriptor/%2Ftest.cwl.json`

For example, the following lists the files including the test json for CWL
https://dockstore.org:8443/api/ga4gh/v2/tools/quay.io%2Fpancancer%2Fpcawg-sanger-cgp-workflow/versions/2.0.3/CWL/files
then you should be able to get the test file with this (it currently doesn't work)
https://dockstore.org:8443/api/ga4gh/v2/tools/quay.io%2Fpancancer%2Fpcawg-sanger-cgp-workflow/versions/2.0.3/CWL/descriptor/%2Ftest1.json

File: dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java [hunk 2]
```diff
@@ -138,6 +138,25 @@ private void toolsIdVersionsVersionIdTypeDescriptorRelativePathNormal() throws E
...
+        String responseObject = response.readEntity(String.class);
+        assertThat(response.getStatus()).isEqualTo(200);
+        assertThat(responseObject.equals("potato"));
+        Response response2 = client.target(basePath + "tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/PLAIN_TEST_CWL_FILE/descriptor/%2Ftest.potato.json").request().get();
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java]: tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/PLAIN-CWL/descriptor/%2Ftest.potato.json here

File: dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java [hunk 3]
```diff
@@ -138,6 +138,25 @@ private void toolsIdVersionsVersionIdTypeDescriptorRelativePathNormal() throws E
...
+        assertThat(response2.getStatus()).isEqualTo(204);
+        Response response3 = checkedResponse(
+            basePath + "tools/quay.io%2Ftest_org%2Ftest6/versions/fakeName/PLAIN_TEST_WDL_FILE/descriptor/%2Ftest.wdl.json");
```
Comment (denis-yuen) [inline dockstore-integration-testing/src/test/java/io/dockstore/client/cli/GA4GHIT.java]: ditto

**Comment by garyluu** [comment]
Closing in favour of new branch based off hotfix

**Pull request closed without merging**